source/manual/purge-cache.html.md

```diff
@@ -54,18 +54,35 @@ GOV.UK internal varnish.
 
 ## Purging a page from Fastly manually (e.g. if GOV.UK Production is dead)
 
-To purge content on the Fastly cache nodes, SSH onto a `cache` machine and use the
-PURGE method against the URL you wish to purge. For instance:
+To purge content on the Fastly cache nodes, use the PURGE method against the URL
+you wish to purge, passing the correct Fastly API token for the environment.
+
+First, connect to the GDS VPN if you're out of the office or on GovWiFi. Our
+Fastly account is configured with an IP allowlist, preventing access to the
+Fastly website or API from outside the GDS network.
+
+Next, extract the Fastly API token. Check out the `govuk-secrets` repo, install
+dependencies with `bundle install`, `cd` into the `puppet_aws` directory,
+and run the following command, replacing `production` with a different
+environment if required:
+
+```sh
+$ FASTLY_API_TOKEN=$(bundle exec rake 'eyaml:decrypt_value[production,govuk_jenkins::jobs::clear_cdn_cache::fastly_api_token]')
```

Comment:
I like the shorthand here, but it didn't work for me. Echoing the variable, I had empty output.

When I ran the `bundle exec rake ...` command directly, I found out why: I needed to enter the password for my keychain.

The `echo` statement below should help developers catch that, but it might be safer to just say "retrieve the token using `bundle exec ...` and then leaving it up to the developer to set the `$FASTLY_API_TOKEN` alias or just hardcode the token when they do the `curl`.

Reply:
Good point!